views/cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+    "baseUrl": "https://tao-authoring-1.docker.localhost",
```

Comment:
let's move this tot he readme and not commit this please 
having some hard-coded data is not good as a committed file

Reply:
`cypress.json` and `cypress.env.json` were replaced with .sample files.